tools/rna_tools/ribotaper/ribotaper_part1_create_annotation_files.xml

```diff
@@ -87,6 +87,24 @@ exon sequences and transcript structures.
 is used as an input for ``ribotaper part 2``.
 *Annotation_path*
 is used as an input for ``ribotaper part 3``.
+
+Important notes
+----------------
+
+- To install the galaxy tool, please enable conda installation in **galaxy.ini**
+
+- Please add the following line to **datatypes_conf.xml** to enable the data type, **compressed_archive**,
+
+    <datatype extension="compressed_archive" type="galaxy.datatypes.binary:CompressedArchive" subclass="True" display_in_upload="True"/>
```

Comment:
I don't remember exactly why using compressed_archive. I think we talked about it. compressed_archive could the inspration of u